packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -23,7 +25,16 @@ const defaultProps = {
 const Toolbar = ({
   align, children, ...customProps
 }) => {
-  let toolbarClassNames = cx('toolbar', `${align}-align`);
+  const theme = React.useContext(ThemeContext);
+
+  let toolbarClassNames = classNames(
+    cx(
+      'toolbar',
+      `${align}-align`,
+      theme.className,
+    ),
+    customProps.className,
```

Comment:
CustomProps wasn't pass previously. Not sure we should add it now.

Reply:
CustomProps were being passed before but something like [this](https://github.com/cerner/terra-core/pull/3014/files#diff-ff210e5dbf27c1d1bc3b7bcee1f88c3dL27) 

I did miss to make toolbarClassName to be const and forgotten to remove its reassignment. Updated in this commit https://github.com/cerner/terra-core/pull/3014/commits/5095b7040c6c0b0fad6c5126cf956d2de734402c